pkg/ebpf/c/tracee.bpf.c

```diff
@@ -3137,6 +3137,12 @@ int BPF_KPROBE(trace_do_exit)
 SEC("uprobe/trigger_syscall_event")
 int uprobe_syscall_trigger(struct pt_regs *ctx)
 {
+#if defined(__TARGET_ARCH_x86)
```

Comment:
BTW I used `u64 *syscall_table_addr` instead of a `unsigned long *` because I think the value from `get_symboladdr` will be a `* (void *)`, which is u64 (size of a pointer). No ? Just checking..

Reply:
Despite styling changes (to something way less verbose) I think the initial "ifdef" to SYSCALLS_TO_CHECK will save you some ifdefs inside the function.